File: allSpice/Models/Favorite.cs
namespace allSpice.Models;

public class Favorite :IRepoItem<string>
{
  public string Id { get; set; }
  public int recipeId { get; set; }
  public string AccountId { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
}

File: allSpice.client/src/models/Favorite.js
export class Favorite {
  constructor(data) {
    this.id = data.id;
    this.recipeId = data.recipeId;
    this.accountId = data.accountId;
    this.createdAt = data.createdAt;
    this.updatedAt = data.updatedAt;
  }
}
